cmd/kubeadm/app/apis/kubeadm/v1beta1/defaults_unix.go

```diff
@@ -23,6 +23,6 @@ const (
 	DefaultCACertPath = "/etc/kubernetes/pki/ca.crt"
 	// DefaultUrlScheme defines default socket url prefix
 	DefaultUrlScheme = "unix"
-	// DefaultCRISocket defines the default cri socket
-	DefaultCRISocket = "/var/run/dockershim.sock"
+	// DefaultDockerCRISocket defines the default Docker CRI socket
+	DefaultDockerCRISocket = "/var/run/dockershim.sock"
```

Comment:
I think that constants related to CRI socket should be moved to `/cmd/kubeadm/app/constants/constants.go` (or to a new `constants_unix.go`), because this is not anymore part of the versioned API